easybuild/easyconfigs/y/yaff/yaff-1.4.2-intel-2017b-Python-3.6.3.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PythonPackage'
+
+name = 'yaff'
+version = '1.4.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://molmod.github.io/yaff/'
+description = """Yaff stands for 'Yet another force field'. It is a pythonic force-field code."""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+toolchainopts = {'lowopt': True}
+
+source_urls = ['https://github.com/molmod/yaff/releases/download/%(version)s']
+sources = [SOURCE_TAR_GZ]
+patches = ['yaff-1.4.2-add-missing-include.patch']
```

Comment:
Please include `SHA256` checksums for both `sources` and `patches` (pro tip: use `--inject-checksums`).

Also avoid hardcoding the `yaff` version in `patches`, use `%(version)s` instead.